internal/jimm/juju/juju_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2025 Canonical.
+package juju_test
+
+import (
+	"context"
+	"time"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/google/uuid"
+	"github.com/juju/names/v5"
+
+	"github.com/canonical/jimm/v3/internal/db"
+	"github.com/canonical/jimm/v3/internal/jimm/credentials"
+	"github.com/canonical/jimm/v3/internal/jimm/juju"
+	"github.com/canonical/jimm/v3/internal/jimm/permissions"
+	"github.com/canonical/jimm/v3/internal/testutils/jimmtest"
+)
+
+type parameters struct {
+	Dialer          juju.Dialer
+	CredentialStore credentials.CredentialStore
+}
+
+func NewTestJujuManager(c *qt.C, p *parameters) *juju.JIMM {
```

Comment:
Instead of using jimmtest.NewJIMM we now have a simpler NewTestJujuManager function. Reserving `jimmtest.NewJIMM` for use in integration tests (there are a few places where it's still used that I want to get rid of soon).

Reply:
does not need to be exported